packages/api-audit-logs/__tests__/utils/getAuditObject.test.ts

```diff
@@ -0,0 +1,543 @@
+import { getAuditObject } from "~/utils/getAuditObject";
+import { auditLogsApps } from "~/config";
+
+describe("getAuditObject", () => {
+    it("should construct audit object from apps", () => {
```

Comment:
I wrote a test to see what does the `getAuditObject` actually do.
I don't understand why you have a function which transforms a hardcoded object into some other structure. Why didn't you hardcode it immediately? You are not using the original, `auditLogsApps`, object anywhere but for the `getAuditObject` function.